src/components/styled.components/Calender.styled.js

```diff
@@ -1,8 +1,18 @@
 import styled from 'styled-components';
 
 export const CalenderContainer = styled.div`
-	margin: 20px;
-	padding: 20px;
-	border: 3px solid black;
-	border-radius: 20px;
+	margin-top: 30px;
+	margin-left: 30px;
+`;
+
+export const Calender_selected_date = styled.button`
```

Comment:
This name is also quite confusing. I wouldn't think that this is the name for a `button`, but `p` element instead.